endpoints/openrtb2/video_auction.go

```diff
@@ -440,15 +440,15 @@ func getVideoStoredRequestId(request []byte) (string, error) {
 
 func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.BidRequest) error {
 
-	if videoRequest.Site.Page != "" {
-		bidRequest.Site = &videoRequest.Site
+	if videoRequest.Site != nil {
+		bidRequest.Site = videoRequest.Site
 		if &videoRequest.Content != nil {
 			bidRequest.Site.Content = &videoRequest.Content
 		}
 	}
 
-	if videoRequest.App.Domain != "" {
-		bidRequest.App = &videoRequest.App
+	if videoRequest.App != nil {
+		bidRequest.App = videoRequest.App
```

Comment:
Do we still want fo check for the `Domain` string content?
```
450     if videoRequest.App != nil && videoRequest.App.Domain != "" {
451         bidRequest.App = videoRequest.App
452         if &videoRequest.Content != nil {
453             bidRequest.App.Content = &videoRequest.Content
454         }
455     }
456
video_auction.go
```

Reply:
For `domain` here is the definition from Open RTB spec:
```
Attribute | Type  | Description
domain  | string  |  Domain of the app (e.g., “mygame.foo.com”).
```
If we need to add reg exp verification, how it should look like? 
@hhhjort @guscarreon please share how do you guys think it should be.